client/components/language-picker/language-names.js

```diff
@@ -0,0 +1,141 @@
+export default {
+	/* eslint-disable quote-props */
+	'af': translate => translate( 'Afrikaans' ),
+	'als': translate => translate( 'Alemannic' ),
+	'am': translate => translate( 'Amharic' ),
+	'ar': translate => translate( 'Arabic' ),
+	'arc': translate => translate( 'Aramaic' ),
+	'as': translate => translate( 'Assamese' ),
+	'ast': translate => translate( 'Asturian' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Asturias' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).